ddtrace/_trace/tracer.py

```diff
@@ -131,15 +131,17 @@ def _default_span_processors_factory(
                 from ddtrace.appsec._api_security.api_manager import APIManager
 
                 APIManager.enable()
+                asm_config._api_security_active = True
 
             appsec_processor = _start_appsec_processor()
             if appsec_processor:
                 span_processors.append(appsec_processor)
         else:
-            if asm_config._api_security_enabled:
+            if asm_config._api_security_active:
```

Comment:
Maybe add a comment to why we want to keep the state of this: why we need to avoid calling `APIManager.disable()`